### PR TITLE
Bugfix/azuread next page

### PR DESCRIPTION
**Pull request by kalisb**
We are having some large groups in Azure AD with more than 100 members and @odata.nextLink is not getting processed

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 app = Flask(__name__)
 github_app = GitHubApp(app)
 directory = DirectoryClient()
-addUserAsMember = os.environ.get("ADD_MEMBER", False)
+addUserAsMember = os.getenv('ADD_MEMBER', 'False') == 'True'
```
Comment (primetheus) [inline app.py:17]: I didn't catch this in the last PR, but this should go in `githubapp/__init__.py` instead of here

Comment (kalisb) [reply app.py]: What about removing `if org.is_member(user)` - or it is better with env variable specifying if the user should be added to the org?

Comment (primetheus) [reply app.py]: @kalisb it's better to keep it as a variable. There are cases when you _don't_ want to add people to an org, like when you're using GHEC and accidentally fat-finger a user ID, or if you're using a backend like LDAP where a user's email is tied to a different user account on github.com. It should be an opt-in behavior

File: githubapp/__init__.py [hunk 2]
```diff
@@ -37,3 +37,5 @@
     rootlogger.warn('TEST_MODE should be set to "true" or "false"')
     rootlogger.warn(e)
     TEST_MODE = False
+# Check if should add member to organization
+ADD_MEMBER = os.getenv("ADD_MEMBER", "False") == "True"
```
Comment (primetheus) [inline githubapp/__init__.py:41]:
```suggestion
TEST_MODE = strtobool(os.environ.get("ADD_MEMBER", "False"))
```
@kalisb would this be better? I think we'd want to allow for `0|1`, `true|false`, `TRUE|FALSE` or other casing combinations

**Comment by primetheus** [comment]
closes #72

**Comment by primetheus** [comment]
fixes #81

**Pull request merged**